<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskWindow</class>
 <widget class="QMainWindow" name="TaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Learning window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGetItems">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>610</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get items</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listItems">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>301</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbQuizes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quizes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbLessons">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lessons</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelItemList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>List of items</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>839</x>
      <y>129</y>
      <width>431</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelTopic">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Discussion about </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputChat"/>
     </item>
     <item>
      <widget class="QLineEdit" name="inputMessage"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelLoggedUsername">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Currently logged as : </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>501</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnCheckAnswer">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>410</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Check answer</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Title/Question</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelContent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lesson/Answer</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputContent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>481</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLearnet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Learnet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>The network that helps you learn about computer networks</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1297</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
